tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -1291,6 +1291,51 @@ def _verify_process_code_and_output(self, command_result):
         self.assertEqual(command_result.process.returncode, 0)
 
 
+@skipIf(
+    ((IS_WINDOWS and RUNNING_ON_CI) and not CI_OVERRIDE),
+    "Skip build tests on windows when running in CI unless overridden",
+)
+class TestRepeatedBuildHitsCache(BuildIntegBase):
+    # Use template containing both functions and layers
+    template = "layers-functions-template.yaml"
+
+    @parameterized.expand([(True,), (False,)])
+    @skipIf(SKIP_DOCKER_TESTS, SKIP_DOCKER_MESSAGE)
```

Comment:
Can we just move this test under the "cache builds" class we already have?

Reply:
Yes good point :+1:

Reply:
@jfuss You mean to move this test inside the `TestBuildWithCacheBuilds` class?
The problem is that `TestBuildWithCacheBuilds` uses `dedup-functions-template.yaml`, which doesn't have any Layers. So it won't test what we want.

If it's important, I can probably refactor it to use `layers-functions-template.yaml`, and then we can have both tests inside the same class.